src/app/[locale]/seller/registration/page.tsx

```diff
@@ -194,18 +210,63 @@ const SellerRegistrationForm = () => {
   };
 
   return (
-    <>
+    <>    
+      {loading && <div className="loading">Loading...</div>}
```

Comment:
Is this loading section necessary when we have a skeleton preload effect?

Reply:
Yes It is cos the page is loaded first before the data except we are to use partial loading method